src/main/java/com/musala/atmosphere/agent/DeviceChangeListener.java

```diff
@@ -28,24 +26,18 @@ public DeviceChangeListener() {
      * and updates the list of devices on the agent.
      * </p>
      *
-     * @param serverIPAddress
-     *        - IP address of the server's RMI registry
-     * @param serverRmiPort
-     *        - Port on which the server's RMI registry is opened
+     * @param isServerSet
+     *        - whether the server is set
      */
-    public DeviceChangeListener(String serverIPAddress, int serverRmiPort) {
+    public DeviceChangeListener(boolean isServerSet) {
```

Comment:
Change the name to something more meaningful like isConnectedToServer or isServerConnected